src/initializers/log4js/json-appender.ts

```diff
@@ -0,0 +1,42 @@
+import * as lodash from 'lodash';
+
+const jsonAppender = ({ filter_status = [] }) => {
```

Comment:
Do we use the filter_status? It is used in honeybadger appender to not send statuses that are expected eg: 401 unauthorized.

Reply:
Done